optimum/intel/neural_compressor/quantization.py

```diff
@@ -97,6 +100,12 @@ def __init__(
         self.calib_func = calib_func
         if calib_dataloader is not None:
             calib_dataloader = IncDataLoader.from_pytorch_dataloader(calib_dataloader)
+        elif example_inputs is not None:
+            if is_torch_less_than_1_13:
+                raise EnvironmentError("PyTorch must be 1.13 or above.")
```

Comment:
Shouldn't we keep the `calib_dataloader` for post-training static quantization as it was previously required and add the `example_inputs` only for QAT ? I think it's not worth it to add breaking changes just before the big refactorization we are working on. Also the user would need to preprocess the dataset according to the model's input which can be tricky